python/ql/src/semmle/python/Stmts.qll

```diff
@@ -405,22 +407,25 @@ class TemplateWrite extends TemplateWrite_ {
     override Stmt getASubStatement() { none() }
 }
 
+/** An asynchronous `for` statement, such as `async for varname in Expr: ...` */
 class AsyncFor extends For {
     /* syntax: async for varname in Expr: ... */
     AsyncFor() { this.isAsync() }
 }
 
+/** An asynchronous `with` statement, such as `async with varname as Expr: ...` */
 class AsyncWith extends With {
     /* syntax: async with Expr as varname: ... */
     AsyncWith() { this.isAsync() }
 }
 
 /** A list of statements */
 class StmtList extends StmtList_ {
-    /** Whether this list of statements contains s */
+    /** Holds if this list of statements contains the AST node `a` */
     predicate contains(AstNode a) {
         exists(Stmt item | item = this.getAnItem() | item = a or item.contains(a))
     }
 
+    /** Gets the last item in this list of statements. */
```

Comment:
```suggestion
    /** Gets the last item in this list of statements, if any such exists. */
```

Reply:
[Style guide](https://github.com/github/codeql/blob/master/docs/qldoc-style-guide.md#predicates-with-result) says a simple "if any" is preferred, so I will do that.

Reply:
True, that is fine.